test/e2e/_incluster-test-code/memcached_test.go

```diff
@@ -317,8 +319,7 @@ func memcachedMetricsTest(t *testing.T, f *framework.Framework, ctx *framework.T
 		return fmt.Errorf("no labels found in metrics Service")
 	}
 
-	// TODO(lili): Make port a constant in internal/scaffold/cmd.go.
-	response, err := getMetrics(t, f, s.Spec.Selector, namespace, "8383")
+	response, err := getMetrics(t, f, s.Spec.Selector, namespace, strconv.Itoa(int(metrics.PortNumber)))
```

Comment:
Should we change the `getMetrics` signature so that we can just call `getMetrics(t, f, s.Spec.Selector, namespace, metrics.PortNumber)`?